docs/docs/sourcing-from-ghost.md

```diff
@@ -62,43 +62,63 @@ Once the source plugin is set up, you can use the `createPages` API in `gatsby-n
 There are several ways to structure queries depending on how you prefer to work, but here's a very minimal example:
 
 ```javascript:title=gatsby-node.js
+const _ = require(`lodash`)
+const Promise = require(`bluebird`)
+const path = require(`path`)
+
 exports.createPages = ({ graphql, actions }) => {
   const { createPage } = actions
   const createPosts = new Promise((resolve, reject) => {
     const postTemplate = path.resolve(`./src/templates/post.js`)
-
     resolve(
       graphql(`
-        {
-          allGhostPost(sort: { order: ASC, fields: published_at }) {
-            edges {
-              node {
-                slug
-              }
-            }
-          }
+                {
+                    allGhostPost(
+                        sort: {order: ASC, fields: published_at},
+                        filter: {
```

Comment:
This filter isn't needed any more - we used to have dummy data we provided to try to stop null/undefined values causing errors, but we added a different workaround to gatsby-source-ghost in v3.2.0.

I've removed these filters from gatsby-starter-ghost now as well so you can see: https://github.com/TryGhost/gatsby-starter-ghost/commit/4facfb723b97a166c8c310473fae41f3c19b2af3